components/logins/examples/sync_pass_sql.rs

```diff
@@ -229,7 +230,7 @@ fn prompt_record_id(e: &PasswordEngine, action: &str) -> Result<Option<String>>
         log::info!("No such index");
         return Ok(None);
     }
-    Ok(Some(index_to_id[input].clone()))
+    Ok(Some(index_to_id[input].as_str().into()))
```

Comment:
I have left this one as it is here, since it can't be moved out of borrowed.

Reply:
That's fine, the examples aren't as important anyway.